src/util/std_types.h

```diff
@@ -25,6 +25,13 @@ Author: Daniel Kroening, kroening@kroening.com
 class constant_exprt;
 class namespacet;
 
+/// This method tests,
+/// if the given typet is a constant
+inline bool is_constant(const typet &type)
+{
+  return type.id() == ID_constant;
+}
```

Comment:
This makes no sense. If this ever were to be true, we'd have messed something up. Maybe `return type.get_bool(ID_C_constant);`?